grails-app/i18n/messages.properties

```diff
@@ -65,3 +65,10 @@ job.failure.defaultMessage=An unexepected problem has occurred.
 job.error.email=Please supply a valid email address
 job.emailAddress.nullable=Please enter a valid email address so we can notify you when the job is complete.
 job.emailAddress.email.error=Please supply a valid email address
+
+job.id.label=Job Id:
```

Comment:
I don't think the ':'s belong here.  How the label and the 'value' are arranged (which is what I think this is) is a property of the view.  e.g. the separation might be achieved a different way (fonts, spacing etc) but the text stays the same.  Hopefully that makes sense?